### DISCUSSION TITLE
Multid und Dsld müssen bei der 7490 7.51 Labor manuell gestartet werden?!

### DISCUSSION THREAD

**Discussion by adn77** [Freetz-NG/freetz-ng#750] [Allgemein]
Ich bin mir des Eintrags https://freetz-ng.github.io/freetz-ng/NEWS.html#2022-12-11 bewusst...
(Port 53 hat be mir DNSMasq)

Aktuell starte ich daher Multid und Dsld manuell über die Freetz Web-Gui.
Bei einigen Änderungen im Fritz-WebUI (z.B. VPN Einrichten) schalten sich die beiden Daemons ebenfalls ab.

Sollte ich versuchen, Multid in einer while-Schleife in der rc.custom zu starten oder gibt es bessere Ideen?

**Comment by fda77** [comment]
Die letzte Zeile ist am wichtigesten, es sollte also alles wie vorher aus funktinieren.
Es gibt aber konfiguartiuonen in denen avm diese dienste nicht startet.
Dann könnte noch ein werksreset helfen um zu sehen  ob es ein config problem ist

**Reply by adn77** [reply]
Dankeschön!
Werde ich versuchen, sobald die 7.51 final raus ist und keine Besserung eintritt.
